packages/ecc-client-ga4gh-wes/src/components/wes-create-run/definition/wesCreateRun.template.ts

```diff
@@ -0,0 +1,48 @@
+import { html, repeat } from '@microsoft/fast-element';
+import WESCreateRun from './wesCreateRun.js';
+
+const fields = [
+  'workflow_params',
+  'workflow_type',
+  'workflow_type_version',
+  'tags',
+  'workflow_engine_parameters',
+  'workflow_url',
+];
+
+const template = html<WESCreateRun>`
+  <form>
+    ${repeat(
+      () => fields,
+      html`
+        <div class="label-input">
+          ${(x, c) => console.log(x, c)}
+          <label for=${(x) => x}>${(x) => x}</label>
+          <input
+            type="text"
+            id=${(x) => x}
+            name=${(x) => x}
+            @input=${(_, c) => c.parent.handleInput(c.event)}
+            required
+          />
+        </div>
+      `
+    )}
+    <div class="label-input">
+      <label for="workflow_attachment">workflow_attachment</label>
+      <input
```

Comment:
the api specifies no accepted file types?

Reply:
The parameter type is `formData` and Data type is `Array[string]` :+1: